app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -34,6 +36,16 @@ class MS.Views.TimelineTableView extends Backbone.View
     'mouseenter .step-end-date': 'showDependentSteps'
```

Comment:
We still want these two functions to work as before with the yellow highlighting:

```
'mouseenter .step-start-date': 'showPrecedentStep'
'mouseenter .step-end-date': 'showDependentSteps'
```

Maybe they won't be called here, but instead will be called from inside the `highlightStep` function. But the functionality needs to stay.